### PR TITLE
Add option to exclude unconfirmed UTXOs at TXP creation

### PR DESCRIPTION
**Pull request by isocolsky**

### CODE REVIEW

File: lib/server.js [hunk 1]
```diff
@@ -687,16 +679,15 @@ WalletService.prototype.getUtxos = function(cb) {
...
-  balance.totalAmount = Utils.strip(_.reduce(utxos, function(sum, utxo) {
-    return sum + utxo.satoshis;
-  }, 0));
-
-  balance.lockedAmount = Utils.strip(_.reduce(_.filter(utxos, {
-    locked: true
-  }), function(sum, utxo) {
-    return sum + utxo.satoshis;
-  }, 0));
+  var balance = {
```
Comment (matiu) [inline lib/server.js]: nice!

File: test/integration/server.js [hunk 2]
```diff
@@ -289,11 +288,11 @@ helpers.createAddresses = function(server, wallet, main, change, cb) {
 
 var storage, blockchainExplorer;
 
-var useMongo = false;
+var useMongoDb = !!process.env.USE_MONGO_DB;
```
Comment (matiu) [inline test/integration/server.js]: :+1:

**Comment by matiu** [comment]
Great job!

**Comment by isocolsky** [comment]
Thanks for the review!

**Pull request merged**